frontend/src/components/CodingChallengeCard.tsx

```
@@ -17,8 +17,14 @@ export default function ConceptReviewCard({
     const handleClick = () => {
         router.push(`/learn/challenge/${module_id}`);
     };
+    // TODO: making this paid is an issue because I changed the fetch code to use the open tag, which doesn't account for being unlocked
+        // I would need to change it to send the ones for the open units and then not open the modules when adding them for the user
     return (
-        <div className={styles.challengeCard} onClick={handleClick} role="button">
+        <div
+            className={styles.challengeCard}
+            onClick={handleClick}
+            role="button"
+        >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)